assets/javascripts/discourse/templates/components/realtime-validation-settings.hbs

```diff
@@ -0,0 +1,32 @@
+<div>
+    <label>{{i18n 'admin.wizard.field.validations.header'}}</label>
+</div>
+{{#each-in field.validations as |name props|}}
```

Comment:
@fzngagan You can probably guess I'm going to say that this UI needs more work. Here's some issues: 

1) there's no way to add additional validations, despite there being ``each-in`` here. I suggest just remove the list structure and restrict it to one validation per field for now. If you're adding functionality it should be usable. 

2) We've discussed how "similar topics" is one type amongst multiple possible types. You need to make that input a combo-box to support multiple types even if there is only one type for now, not an input. Same with the ``props.postion``. 

You need to think about the "structure" we want to buid here and make the UI reflect that even if there's only one specific implementation of the structure for now. One way to structure the UI would be to have three hoirzonal sections representing the three types of inputs here, e.g. 

<img width="523" alt="Screenshot at Feb 08 21-20-22" src="https://user-images.githubusercontent.com/5931623/107207221-283fd080-6a54-11eb-958f-e34e33d1d1b6.png">

Presentation matters. This should "feel" like it belongs with the other custom wizard settings. currently it doesn't.

Reply:
Style Tweaks, and hide these settings under advanced. Name it as Enable Similar Topics

Reply:
Done